.quiet_quality.ci-cd.yml

```diff
@@ -0,0 +1,8 @@
+---
+executor: concurrent
+comparison_branch: origin/main
+changed_files: true
+annotator: github_stdout
+filter_messages: true
```

Comment:
I think since we're currently green on these checks, and the project is small, it might be best to just run all-files and unfiltered? It _shouldn't_ make a difference, but it's easier to reason about.

Oh, that reminds me that I should require update-before-merge on these projects!